.travis.yml

```diff
@@ -44,7 +40,8 @@ sudo: false
 install:
 - travis_retry npm install
 script:
-- if [[ -z "$INTEGRATION_TEST" && -z "$DISABLE_TESTS" ]]; then npm test; fi
+- npm i -g npm
```

Comment:
Why is that needed?

Reply:
I guess it was added becasue we're using `npm` in the `packaging` test in child processes. Not sure if a global install is necessay though.

Reply:
removed